packages/compiler/src/core/compiler-code-fixes/triple-quote-indent.codefix.ts

```diff
@@ -0,0 +1,74 @@
+import { splitLines } from "../../formatter/print/printer.js";
+import { isWhiteSpaceSingleLine } from "../charcode.js";
+import { defineCodeFix, getSourceLocation } from "../diagnostics.js";
+import type { DiagnosticTarget } from "../types.js";
+
+export function createTripleQuoteIndentCodeFix(diagnosticTarget: DiagnosticTarget) {
+  return defineCodeFix({
+    id: "triple-quote-indent",
+    label: "Format triple-quote-indent",
+    fix: (context) => {
+      const location = getSourceLocation(diagnosticTarget);
+      const splitStr = "\r\n";
+      const newlineRegex = /(\r\n|\n|\r)/gm;
+      const tripleQuote = '"""';
+      const tripleQuoteLen = tripleQuote.length;
+      const text = location.file.text.slice(
+        location.pos + tripleQuoteLen,
+        location.end - tripleQuoteLen,
+      );
+
+      const lines = splitLines(text);
```

Comment:
how about replace the newline chars here so that you dont need to worry about it in other places later?